index.yaml

```diff
@@ -3005,12 +3268,11 @@ entries:
       catalog.cattle.io/namespace: cattle-monitoring-system
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: monitoring.coreos.com.prometheus/v1
-      catalog.cattle.io/rancher-version: < 2.6.1-0
```

Comment:
I think we should not drop the annotaoin in this PR, right?

Reply:
The drop happens in multiple places& charts.

Reply:
@aiyengar2 is this addressed by https://github.com/rancher/charts/pull/1520?

Reply:
Your PR removes much more than the above PR adds back.

Reply:
Well, only for that field though

Reply:
Other than the annotation being dropped here (and wherever it exists), the PR is updating everything in place in `index.yaml` or is adding the new charts/assets. That's the only issue I see now, is that the same for you?

Reply:
that is right. All newly-added charts look right; the only issue is the drop of annotations on exiting charts